components/CheckInOutForm/index.tsx

```diff
@@ -0,0 +1,128 @@
+import {
+  Autocomplete,
+  Box,
+  FormControl,
+  IconButton,
+  TextField,
+  Typography,
+} from '@mui/material'
+import { DateTimePicker } from '@mui/x-date-pickers'
+import dayjs, { Dayjs } from 'dayjs'
+import React from 'react'
+import { ItemDefinition, User } from '../../utils/types'
+import AddOutlinedIcon from '@mui/icons-material/AddOutlined'
+import RemoveOutlinedIcon from '@mui/icons-material/RemoveOutlined'
+import colors from '../../utils/colors'
+
+interface Props {
+  kioskMode: boolean
+  users: User[]
+  itemDefinitions: ItemDefinition[]
+}
+
+function CheckInOutForm({ kioskMode, users, itemDefinitions }: Props) {
+  const [date, setDate] = React.useState<Dayjs | null>(dayjs(new Date()))
+  const [quantity, setQuantity] = React.useState<number>(1)
+  const [selectedStaff, setSelectedStaff] = React.useState<User | null>()
+  const [selectedItemDefinition, setSelectedItemDefinition] =
+    React.useState<ItemDefinition | null>()
+
+  return (
+    <>
+      <FormControl fullWidth>
+        {/* Staff member, Date, and Item input fields */}
+        {kioskMode && (
+          <Autocomplete
+            options={users.map((user) => user.name)}
+            renderInput={(params) => (
+              <TextField {...params} label="Staff Member" />
+            )}
+            onChange={(_e, name) =>
+              setSelectedStaff(users.find((user) => name === user.name))
```

Comment:
ah i see, i still think we would want to have the id instead since im pretty sure thats what gets passed to mongo (not sure tho)

Reply:
I set the val as the User object just for flexibility in case we want/need to do something else with the rest of the user information. When it's submitted, it'd be easy enough to just do `selectedStaff._id`